--- app/etc/env.php	2019-08-26 22:21:59.449713000 +0200
+++ app/etc/env_new.php	2019-08-26 22:43:41.253935000 +0200
@@ -34,12 +34,40 @@
     'cache' => [
         'frontend' => [
             'default' => [
-                'id_prefix' => '40d_'
+                'backend' => 'Cm_Cache_Backend_Redis',
+                'backend_options' => [
+                    'server' => 'redis',
+                    'port' => '6379',
+                    'persistent' => '',
+                    'database' => '0',
+                    'force_standalone' => '0',
+                    'connect_retries' => '1',
+                    'read_timeout' => '10',
+                    'automatic_cleaning_factor' => '0',
+                    'compress_data' => '1',
+                    'compress_tags' => '1',
+                    'compress_threshold' => '20480',
+                    'compression_lib' => 'gzip',
+                ],
             ],
             'page_cache' => [
-                'id_prefix' => '40d_'
-            ]
-        ]
+                'backend' => 'Cm_Cache_Backend_Redis',
+                'backend_options' => [
+                    'server' => 'redis',
+                    'port' => '6379',
+                    'persistent' => '',
+                    'database' => '1',
+                    'force_standalone' => '0',
+                    'connect_retries' => '1',
+                    'read_timeout' => '10',
+                    'automatic_cleaning_factor' => '0',
+                    'compress_data' => '0',
+                    'compress_tags' => '1',
+                    'compress_threshold' => '20480',
+                    'compression_lib' => 'gzip',
+                ],
+            ],
+        ],
     ],
     'lock' => [
         'provider' => 'db',
